social-network/src/Author.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default class Author extends React.Component {
+  render() {
+    const author = this.props.author;
+    const date = new Date(this.props.postedDate);
+    const minutes = this.props.minutesToRead;
+    const dateFormat = new Intl.DateTimeFormat('en-US', { month: 'short', day: 'numeric' });
+    const isMember = author.isMediumMember;
+
+    return <div className="author">
+      <div className="author-image">
+        <svg viewBox="0 0 40 40" xmlns="http://www.w3.org/2000/svg" className={isMember ? "author-image-frame" : "hidden"}>
```

Comment:
SVGs galore. If you ever work with larger SVGs, they can also be imported similar to the logo svg in the create-react-app starter.